package.json

```diff
@@ -3,9 +3,10 @@
   "description": "A library for writing modern OpenTripPlanner-compatible multimodal journey planning web applications using React and Redux",
   "main": "build/index.js",
   "scripts": {
+    "a11y-test": "mastarm test a11y --force-exit",
     "build": "mastarm build --env production",
     "cover": "mastarm test -e test --coverage",
-    "jest": "mastarm test -e test",
+    "jest": "mastarm test -e test __tests__",
     "lint": "mastarm lint lib __tests__ --quiet",
```

Comment:
Please also lint the `a11y` directory.

Reply:
Added in 6188d3ac981359220491cc83995fe7f351d9efb2